src/app/theme/sass/_form.scss

```diff
@@ -1,3 +1,5 @@
+@import "../components/baCheckbox/baCheckbox";
```

Comment:
@Tibing I'm pretty sure it's better to include the component styles somewhere where we actually use it, for example in page component scss file.
Thus, if user doesn't need the checkbox component he won't have its css loaded globally.